src/includes/performance/enable-manual-instrumentation/java.spring-boot.mdx

```diff
@@ -40,14 +40,14 @@ import io.sentry.spring.tracing.SentryTransaction
 class ScheduledJob {
 
   @Scheduled(...)
-  @SentryTransaction
+  @SentryTransaction("task")
   fun execute() {
     ...
   }
 }
 ```
 
-`@SentryTransaction` can be configured with custom `name` and `op` properties. If not defined, `name` will be resolved from the class, and the method name.
+`@SentryTransaction` can be configured with custom `name`. If not defined, `name` will be resolved from the class, and the method name.
```

Comment:
isn't `operation` mandatory for the annotation too?

Reply:
It is mandatory and it's in the sample. Here it's about the optional parameters.

Reply:
I still believe it should be somewhere documented what "name", "operation" and "description" are. In some place that is shared between all SDKs. Maybe it's just me, but it's very unclear what goes where in the UI until you try it out.

Reply:
I understand, but the text says that the `name` will be resolved, what's about the `operation`?
either we infer the `operation` automatically too or our samples should be like `@SentryTransaction(name = "name", operation = "task")`, otherwise, if missing the `operation`, it'd throw I believe?

Reply:
or `@SentryTransaction("task")` actually means `operation` == `task`?
maybe we should name the annotations?
`@SentryTransaction(operation = "task")` so it'd be easier to read?

Reply:
Good point.